### DISCUSSION TITLE
Image Sop class supported in fo-dicom

### DISCUSSION THREAD

**Discussion by Ealfaro41** [fo-dicom/fo-dicom#1345] [Q&A]
I bother you to ask where I can get a list of the Image sop classes supported by fo-dicom for receiving and storing images? Any information you can give me will be of great help. Greetings

**Comment by gofal** [comment]
out of the box, fo-dicom supports all image sop classes, where "supporting" means, that the services in fo-dicom can retrieve and store them. If you are implementing a store-scp service, then you have to filter out the image, that you will be able to process afterwards. If you are only storing and sending data, then you can just accept every sop class. If you are implementing a viewer, then i would for example recommend to accept only the ForPresentation sop classes and to reject the ForProcessing sop classes. etc. So which sop classes to "support" is completley up to your application that you are devoloping.
